### DISCUSSION TITLE
[FR] streamer function for the AI Assistant?

### DISCUSSION THREAD

**Discussion by beesinblenders** [chhoumann/quickadd#682] [Ideas]
I'm a fan of the AI Assistant, I found that I didn't need any other text generation plug-in. The only downside for me is to have to wait for the model's entire response before I can read it.

Would it be doable to add the ability to stream responses as they arrive? I think another plug-in does it (Text Generator maybe).

If it's too much trouble to write the streamed response in the note, it could be streamed to  a temporary window and then inserted into the active note when it's all there.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

This ended up tracked as #1030. I did look into streaming - I built it out in another project to see how it feels - and streaming into a note just didn't give a good enough experience to justify the complexity, so I closed that issue as not planned for now. If that changes, #1030 is where it will be revisited.

Glad the AI Assistant is otherwise doing the job for you, and thanks for the suggestion.
